### ISSUE TITLE
ExtRocketMQConsumerConfiguration#selectorExpression not resolve placeholders

### ISSUE THREAD

**Issue by HassanChiang** [apache/rocketmq-spring#376]
**BUG REPORT**

1. Please describe the issue you observed:

- What did you do (The steps to reproduce)?
I set `selectorExpression` value by SpEL: 

```
@ExtRocketMQConsumerConfiguration(
      group = "xxx",
      topic = "xxx",
      selectorExpression = "${spring.profiles.active}")
```

- What did you expect to see?
The `selectorExpression` value should be `dev`.

- What did you see instead?
The `selectorExpression` value is `${spring.profiles.active}`.

2. Please tell us about your environment:
rocketmq-spring 2.2.0
spring.profiles.active=dev

3. Other information (e.g. detailed explanation, logs, related issues, suggestions how to fix, etc):

**Comment by RongtongJin** [comment]
Good catch! Could you submit a pull request to fix the issue?

**Comment by RongtongJin** [comment]
Fixed in #383
